engine/player/unique_gear_dragonflight.cpp

```diff
@@ -3886,6 +3886,111 @@ void neltharions_call_to_chaos( special_effect_t& effect )
   new dbc_proc_callback_t( effect.player, effect );
 }
 
+void neltharions_call_to_dominance( special_effect_t& effect )
+{
+  auto stacking_buff = buff_t::find( effect.player, "domineering_arrogance" );
+
+  if ( !stacking_buff )
+  {
+    stacking_buff = create_buff<buff_t>( effect.player, "domineering_arrogance", effect.player->find_spell( 411661 ) )
+                                                       ->set_default_value( effect.driver()->effectN( 1 ).base_value() );
+  }
+
+  effect.custom_buff = stacking_buff;
+
+  auto stat_effect = new special_effect_t( effect.player );
+
+  // populate stat effect properties here
+
+  auto stat_buff = buff_t::find( effect.player, "call_to_dominance" );
+
+  if ( !stat_buff )
+  {
+    stat_buff = create_buff<buff_t>( effect.player, "call_to_dominance", effect.player->find_spell( 403380 ) )
+                      ->set_stack_change_callback( [ stacking_buff ]( buff_t* b, int prev, int cur )
+                                                   {
+                                                     if ( cur > prev )
+                                                     {
+                                                       b->current_value = stacking_buff->check_stack_value();
+                                                       stacking_buff->expire();
+                                                     }
+                                                   } );
+  }
+
+  stat_effect->custom_buff = stat_buff;
+  stat_effect->type = SPECIAL_EFFECT_EQUIP;
+  stat_effect->source = SPECIAL_EFFECT_SOURCE_ITEM;
+
+  std::set<int> proc_spell_id;
+  int driver_id = effect.spell_id;
+
+  switch ( effect.player->specialization() )
+  {
+    case SHAMAN_ELEMENTAL:
+      driver_id = 408259;
+      proc_spell_id = { { 198067, 192241 } }; // Fire Elemental and Storm Elemental
+      break;
+    case SHAMAN_ENHANCEMENT:
+      driver_id = 408259;
+      proc_spell_id = { { 51533 } }; // Feral Spirit
+      break;
+    case SHAMAN_RESTORATION:
+      driver_id = 408259;
+      proc_spell_id = { { 98008, 108280, 16191 } }; // Spirit Link, Healing Tide, Mana Tide totems
+      break;
+    case MONK_BREWMASTER:
+      driver_id = 408260;
+      proc_spell_id = { { 132578 } }; // Invoke Niuzao
```

Comment:
Call to Arm's Invoke Niuzao is proccing the buff. 
Trigger spell: Weapons of Order: 387184 
Call to Arms talent: 397251, 
Buff spell that summons Niuzao: 395267